src/i_sdlsound.c

```diff
@@ -89,6 +89,10 @@ int use_libsamplerate = 0;
 
 float libsamplerate_scale = 0.65f;
 
+// DWF 2016-05-29
```

Comment:
Remove the signature comment here. It is clutter and made redundant thanks to Git